examples/ssl-factory/src/org/apache/cassandra/security/KubernetesSecretsPEMSslContextFactory.java

```diff
@@ -135,12 +137,12 @@ public KubernetesSecretsPEMSslContextFactory(Map<String, Object> parameters)
         super(parameters);
 
         pemEncodedKey = getValueFromEnv(getString(PEMConfigKey.PRIVATE_KEY_ENV_VAR, DEFAULT_PRIVATE_KEY_ENV_VAR_NAME),
-                                        DEFAULT_PRIVATE_KEY);
```

Comment:
there are all these instances of changed formatting, can we return it back as it was? it happens in the below class too

Reply:
ack. I seem to have the settings issue again. Let me see. I did run ant generate-idea-files before I started to make changes.